src/libraries/System.Collections/src/System/Collections/Generic/SortedDictionary.cs

```diff
@@ -353,7 +353,7 @@ ICollection IDictionary.Values
                     throw new ArgumentNullException(nameof(key));
                 }
 
-                if (value == null && !(default(TValue) == null))
```

Comment:
@stephentoub any reason to assume this form is intentional? Could the change break any intended optimization?

Reply:
They're indistinguishable:
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBLANgHwAEAmARgFgAoQgZgAIS6BhAHgBUA+Ogbyrv4b1gECLjoBZUgAoAlHQC8XACYwAZgEMArrgxS2cgITy6AO224A3HwG06w0ROKyFXA1JUbtu/QuNncuDJWlAC+QA===
I don't know why it was written this way, but both forms are equivalent.